src/packages/excalidraw/README.md

```diff
@@ -0,0 +1,286 @@
+        Update Scene
+      </button>
+      <button
+        className="reset-scene"
+        onClick={() => {
+          excalidrawRef.current.resetScene();
+        }}
+      >
+        Reset Scene
+      </button>
+      <div className="excalidraw-wrapper">
+        <Excalidraw
+          ref={excalidrawRef}
+          width={width}
+          height={height}
+          initialData={InitialData}
+          onChange={onChange}
+          user={{ name: "Excalidraw User" }}
+          onPointerUpdate={(payload) => console.log(payload)}
+        />
+      </div>
+    </div>
+  );
+}
+```
+
+[![Edit excalidraw](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/excalidraw-ehlz3?fontsize=14&hidenavigation=1&theme=dark)
+
+### Props
+
+| Name                                          | Type                                                                                                                                                                                                                                                                                                                           | Default            | Description                                                                                                                                                |
+| --------------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------------ | ---------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [`width`](#width)                             | Number                                                                                                                                                                                                                                                                                                                         | window.innerWidth  | The width of Excalidraw component                                                                                                                          |
+| [`height`](#height)                           | Number                                                                                                                                                                                                                                                                                                                         | window.innerHeight | The height of Excalidraw component                                                                                                                         |
```

Comment:
`window.innerHeight`, `window.innerWidth`, in general anything that is code should be code